core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -462,39 +461,8 @@ public static DataOutputStream getDataOutputStreamUsingAppend(String path, FileT
    * @throws IOException
    */
   public static void truncateFile(String path, FileType fileType, long newSize) throws IOException {
-    path = path.replace("\\", "/");
```

Comment:
why remove these code.

Reply:
want to use the interface 'CarbonFile.truncate' to truncate file uniformly.